app/controllers/peer_reviews_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PeerReviewsController < ApplicationController
+  before_action :set_peer_review, only: [:show, :edit, :update, :destroy]
```

Comment:
Please put in an "authenticate only for admin" check before each of these (see other controllers)